README.md

```diff
@@ -15,12 +15,27 @@ Useful interpolated functions for [styled-components](https://github.com/styled-
 Play with it on [WebpackBin](https://www.webpackbin.com/bins/-Kel3KgddZSrD5oK0fIk)
 
 ```jsx
-import styled from 'styled-components'
-import { prop, ifProp } from 'styled-tools'
+import styled, { css } from 'styled-components'
+import { prop, ifProp, switchProp } from 'styled-tools'
 
 const Button = styled.button`
   color: ${prop('color', 'red')};
   font-size: ${ifProp({ size: 'large' }, '20px', '14px')};
+
+  ${switchProp('type', {
```

Comment:
Let's change `type` for another name, like `theme`, since `type` is a valid html attribute for `button` and might cause conflict.